ThunderRequest/Request.swift

```diff
@@ -15,32 +15,32 @@ import os.log
 public class Request {
 
     /// The base URL for the request e.g. "https://api.mywebsite.com"
-    var baseURL: URL
+    public var baseURL: URL
```

Comment:
Is `public` needed now?

Reply:
Yeah it is because this isn't on an extension 😊 Blood uses this in the error handling logic (via the notification centre notifications ThunderRequest sends) to make sure the base url wasn't for an authentication endpoint.